gensrc/thrift/FrontendService.thrift

```diff
@@ -621,6 +625,20 @@ struct TStreamLoadMultiTablePutResult {
     3: optional list<PaloInternalService.TPipelineFragmentParams> pipeline_params
 }
 
+// StreamLoadWith request status
+struct TStreamLoadWithLoadStatusRequest {
+    1: required Types.TUniqueId loadId
```

Comment:
Do not use `required`, use `optional`.
Same for other fields below.

Reply:
done